bifrost_min_read_check/launcher.py

```diff
@@ -3,139 +3,169 @@
 Launcher file for accessing dockerfile commands
 """
 import argparse
+from io import FileIO, StringIO
 import json
-import subprocess
 import os
 import sys
 import traceback
+import subprocess
 from bifrostlib import datahandling
+from bifrostlib.datahandling import Component
+from bifrostlib.datahandling import ComponentReference
+import yaml
+import pprint
+from typing import List, Dict
+
+from yaml.tokens import StreamEndToken
+
+
+global COMPONENT
+
+
+def initialize():
```

Comment:
Similar comment with this file as datadump. If there is common stuff across all components moving it to the library would be a good idea. Updating all components on common code could be a pain